litellm/llms/vertex_httpx.py

```diff
@@ -845,9 +849,9 @@ def _get_token_and_url(
             endpoint = "generateContent"
             if stream is True:
                 endpoint = "streamGenerateContent"
-                url = f"https://{vertex_location}-aiplatform.googleapis.com/v1/projects/{vertex_project}/locations/{vertex_location}/publishers/google/models/{model}:{endpoint}?alt=sse"
+                url = f"https://{vertex_location}-aiplatform.googleapis.com/v1beta1/projects/{vertex_project}/locations/{vertex_location}/publishers/google/models/{model}:{endpoint}?alt=sse"
```

Comment:
Hey @Manouchehri, why is this change necessary?

Reply:
got it - can we use a flag for this? 

Google recommends using Vertex ai for prod

Google says v1beta endpoints can change without notice